scrapper.py

```diff
@@ -44,18 +83,69 @@ def get_search_urls(self):
         pass
 
 
+class HTMLParser:
+    def __init__(self, article_url, article_id):
+        self.article_url = article_url
+        self.article_id = article_id
+        self.article = Article(article_url, article_id)
+
+    def parse(self):
+        self.article = Article(self.article_url, self.article_id)
+
+        headers = {
+            'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64)AppleWebKit/'
+                          '537.36 (KHTML, like Gecko) Chrome/99.0.4844.51 Safari/537.36',
+            'accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/'
+                      'avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9',
+            'accept-encoding': 'gzip, deflate, br',
+            'accept-language': 'ru,ru-RU;q=0.9,en-US;q=0.8,en;q=0.7,la;q=0.6'}
+
+        response = requests.get(self.article_url, headers)
+        article_bs = BeautifulSoup(response.text, 'lxml')
+
+        # self._fill_article_with_text(article_bs)
+        # self.article.save_raw()
+        return self.article
+
+    def _fill_article_with_text(self, article_bs):
+        pass
+
+
 def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    try:
+        os.rmdir(base_path)
+    except FileNotFoundError:
+        os.mkdir(base_path)
 
 
 def validate_config(crawler_path):
     """
     Validates given config
     """
-    pass
+    with open(crawler_path, 'r', encoding='utf-8') as f:
+        config = json.load(f)
+
+    seed_urls = config["seed_urls"]
+    max_articles = config["total_articles_to_find_and_parse"]
+
+    for url in seed_urls:
+        right_url = re.match(r'https?://', url)
+        if not right_url:
```

Comment:
are you going to accept URLs such as https://github.com/fipl-hse/2021-2-level-ctlr/pull/30/files?

Reply:
and again - re-use the domain name (even for creation of regex - if you prefer)